docs/sources/configure/_index.md

```diff
@@ -1975,6 +1975,10 @@ congestion_control:
 # CLI flag: -store.max-parallel-get-chunk
 [max_parallel_get_chunk: <int> | default = 150]
 
+# Max parallelism used by a table operation.
+# CLI flag: -store.max-parallelism-table-ops
+[max_parallelism_table_ops: <int> | default = 50]
```

Comment:
To be consistent with `max_parallel_get_chunk` we should consider naming it like:

```suggestion
[max_parallel_table_ops: <int> | default = 50]
```